### ISSUE TITLE
[Issue]: The native build on Ubuntu Jammy (22.04 LTS) cannot be finished

### ISSUE THREAD

**Issue by STibor** [jellyfin/jellyfin#9316]
### Please describe your bug

I have tried to build the latest version of Jellyfin (10.8.9) on Ubuntu Jammy but I have got a failure during the build process: "cannot find library liblttng-ust.so.0 needed by  debian/jellyfin-server/usr/lib/jellyfin/bin/libcoreclrtraceptprovider.so".
The build process had stopped and hadn't finished the job at that point.
I have already installed the named package "liblttng-ust" but it did not help at all.
What is the wrong? What have I missed? 

### Jellyfin Version

10.8.0

### if other:

10.8.9

### Environment

```markdown
- OS: Ubuntu Jammy
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bugfixin** [comment]
Shot in the dark here, but is liblttng-ust-dev installed as well?

**Comment by STibor** [comment]
Thx for helping, but unfortunately it did not bring me the solution.

**Comment by STibor** [comment]
Thanks, I figured it out too. I was capable to build at last, but for my sandess I have explored the new version of Jellyfin (10.8.9) does not solve the problem of question for navite folder view. I have arranged my multimedia data in a well declared structure of folders but any of  data view in Jellyfin is unsuitable for viewing this structure. I am honestly the data view in Jellyfin is horrific. It would be a great multimedia server and even it works fine with my Marantz ND8006 but the lack of the folder view makes it unusable. I think this problem will not solve anymore because it would be too big development for a free software and nobody will undertake.
